engine/src/xcc.rs

```diff
@@ -19,7 +19,7 @@ pub const CODE_KEY: &[u8] = b"router_code";
 pub const VERSION_UPDATE_GAS: NearGas = NearGas::new(5_000_000_000_000);
 pub const INITIALIZE_GAS: NearGas = NearGas::new(15_000_000_000_000);
 pub const UNWRAP_AND_REFUND_GAS: NearGas = NearGas::new(25_000_000_000_000);
-pub const WITHDRAW_GAS: NearGas = NearGas::new(30_000_000_000_000);
+pub const WITHDRAW_GAS: NearGas = NearGas::new(40_000_000_000_000);
```

Comment:
This change was needed now that more gas is consumed by the withdraw to Near precompile in the case that it needs a callback (either because of the auto-unwrap or because of the error_refund feature).

This change makes everything work, but it makes me realize that there is a more involved change that should be made. Effectively we are duplicating the auto-unwrap logic in both XCC and in the exit precompile now (XCC always auto-unwrapped wNEAR, now the exit precompile does so too). We could remove that duplication by adding `:unwrap` to the exit message and removing the unwrapping logic from XCC itself. That should probably be done as a separate PR though.

Reply:
I'm surprised that the change doesn't break any test 🤔